keyboards/meetlab/kafka68/info.json

```
@@ -0,0 +1,101 @@
+{
+    "manufacturer": "lucky_studio",
+    "keyboard_name": "kafka68",
+    "maintainer": "yuezp",
+    "development_board": "bluepill",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
```

This is generally preferred to be enabled if possible.
```suggestion
        "extrakey": true,
        "mousekey": true,
```

---

thank you！